src/Advection/centered_reconstruction.jl

```diff
@@ -8,14 +8,19 @@ struct Centered{N, FT, CA} <: AbstractCenteredAdvectionScheme{N, FT}
     Centered{N, FT}(buffer_scheme::CA) where {N, FT, CA} = new{N, FT, CA}(buffer_scheme)
 end
 
-function Centered(FT::DataType=Oceananigans.defaults.FloatType; order = 2)
+function Centered(FT::DataType=Oceananigans.defaults.FloatType; 
+                  order = 2,
+                  buffer_scheme = nothing)
+
     mod(order, 2) != 0 && throw(ArgumentError("Centered reconstruction scheme is defined only for even orders"))
 
     N  = Int(order ÷ 2)
-    if N > 1
-        buffer_scheme = Centered(FT; order=order-2)
-    else
-        buffer_scheme = nothing
+    if isnothing(buffer_scheme)
```

Comment:
you might want to come up with a name for this scheme. Because I feel like "buffer_scheme=nothing" has an actual meaning 🤔